nestkernel/source_table_position.h

```diff
@@ -117,6 +128,12 @@ SourceTablePosition::is_at_end() const
   }
 }
 
+inline void
+SourceTablePosition::decrease()
+{
+  --lcid;
```

Comment:
I think `SourceTablePosition` should guarantee that `lcid >= -1`, so that in `SourceTable`, we can be sure that `x.lcid + 1 >= 0`. So I'd suggest to add `assert( lcid >= -1 )` after the reduction.

Reply:
good point!